xml/ha_config_cli.xml

```diff
@@ -902,7 +902,15 @@ Membership information
    <link xlink:href="http://crmsh.nongnu.org/crm.8.html#cmdhelp_corosync"/>.
   </para>
  </sect1>
- <sect1 xml:id="sec.ha.config.crm.global">
+ <!--taroth 2018-03-14: commenting because example has become useless meanwhile:
+   * no-quorum-policy=stop is now default, ignore should no longer be used
```

Comment:
no-quorum-policy=stop has always been the default actually. Indeed, ignore should no longer be used.

Reply:
Right, adjusted the comment accordingly.